src/main/io/serial.c

```diff
@@ -144,6 +144,7 @@ void pgResetFn_serialConfig(serialConfig_t *serialConfig)
 
     serialConfig->reboot_character = 'R';
     serialConfig->serial_update_rate_hz = 100;
+    serialConfig->softSerialHalfDuplexReset = 1;
```

Comment:
Should be `true`.

Reply:
hah I had it that way but changed it.  will do that now.